tests/testthat/test-shinytest2-tm_a_regression.R

```diff
@@ -0,0 +1,64 @@
+testthat::test_that("e2e - tm_a_regerssion: verify encoding values and widgets", {
```

Comment:
Very ambiguous description of a test containing multiple expectations. This test includes:
- data extract spec elements are initialized with the default values specified by response and regressor args
- plot type has 7 choices
- changing plot type to "Residuals vs Fitted" and "Normal Q-Q" doesn't throw errors
- Outlier definition is hidden by default
- Marking "Display outlier labels" shows Outlier definition panel
- clicking plot settings accordion panel shows plot settings